docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: "3"
+services:
+  solid-server:
+    build: .
+    container_name: solid
+    ports:
+      - "8443:8443"
+    entrypoint: npm run solid start -- --no-reject-unauthorized
```

Comment:
**question** Why is `--no-reject-unauthorized` added? And why is the same(?) thing added as an env variable below as well?

Reply:
Hi @angelo-v 

Because it's exactly same thing that bin/solid-test is doing, as you can see:

# Disable rejectUnauthorized when starting the server
if [ "$COMMAND" == "start" ]; then
  ADD_FLAGS="--no-reject-unauthorized"
  export NODE_TLS_REJECT_UNAUTHORIZED=0
fi

Respectfully speaking and excuse my sincerity, but this is simple docker-compose to help developer (like me) to running up application easily. But it is stuck in here for a long 1 month and ithas been generating so much questions that seems unbelieve to me.. Please feel free to decline.

Reply:
Thanks for the hint @paulorodriguesxv I now understand and I think we can just merge it.

I totally understand your frustration. Unfortunately node-solid-server does not get much attention from the community anymore. Only a tiny group of volunteers keeping an eye on it more or less. I have the permission to merge and will do it, but I am not a NSS maintainer in any regard